src/AsmResolver.DotNet/Cloning/IMemberClonerListener.cs

```diff
@@ -0,0 +1,15 @@
+namespace AsmResolver.DotNet.Cloning
+{
+    /// <summary>
+    /// <see cref="MemberCloner"/> Callback Listener.
+    /// </summary>
+    public interface IMemberClonerListener
+    {
+        /// <summary>
+        /// This function is called for every member got cloned.
```

Comment:
This function is called for every member **that** got cloned.

Reply:
You probably want to fix the same grammar mistake for every member, not just OnClonedMember :p